### PR TITLE
[forge] fix assert image tags have prefix if profile or feature

### PR DESCRIPTION
**Pull request by rustielin**
### Description

When no image tags are provided, make sure we do not assert

Also, only autoselect from forge-big clusters, so we can start to phase out the original (small) forge clusters

### Test Plan

Unit test

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3937)

### CODE REVIEW

**Review by igor-aptos — APPROVED** [review]

**Review by sitalkedia — APPROVED** [review]

File: testsuite/forge.py [hunk 1]
```diff
@@ -992,7 +992,7 @@ def list_eks_clusters(shell: Shell) -> List[str]:
         return [
             cluster_name
             for cluster_name in cluster_result["clusters"]
-            if cluster_name.startswith("aptos-forge-")
+            if cluster_name.startswith("aptos-forge-big-")
```
Comment (perryjrandall) [inline testsuite/forge.py:995]: Lets hardcode this in the run-forge instead of setting this

File: .github/workflows/run-forge.yaml [hunk 2]
```diff
@@ -19,6 +19,7 @@ on:
       FORGE_CLUSTER_NAME:
         required: false
         type: string
+        default: aptos-forge-big-0
```
Comment (perryjrandall) [inline .github/workflows/run-forge.yaml:22]: Instead of setting this here lets unconditionally override in env

File: testsuite/forge.py [hunk 3]
```diff
@@ -1044,12 +1044,14 @@ def last(self, limit: int = 1) -> Generator[str, None, None]:
 
 
 def assert_provided_image_tags_has_profile_or_features(
-    forge_image_tag,
-    image_tag,
+    forge_image_tag: Optional[str],
```
Comment (perryjrandall) [inline testsuite/forge.py:1047]: typing ftw!

File: testsuite/forge.py [hunk 4]
```diff
@@ -1044,12 +1044,14 @@ def last(self, limit: int = 1) -> Generator[str, None, None]:
...
+    image_tag: Optional[str],
     enable_failpoints_feature: bool = False,
```
Comment (perryjrandall) [inline testsuite/forge.py:1049]: are we ever not passing these in? Where are we calling this without these args?

**Review by perryjrandall — APPROVED** [review]
Approving with comment, lets avoid hardcoding in the forge.py itself since that wont take effect until rebase versus hardcode in the run-forge.yaml take effect everywhere

File: testsuite/forge.py (hunk 4 above)
Comment (rustielin) [reply testsuite/forge.py:1049]: they're just None sometimes from `envoption`

Comment (perryjrandall) [reply testsuite/forge.py:1049]: I meant the enable_failpoints_feature optional keyword args

**Pull request merged**